source/includes/_queries.md

```diff
@@ -40,13 +41,22 @@ Each query consists of following things:
     <li> **Query object** might be any of types, described below </li>
```

Comment:
"might be of any type, described below" sounds better in my opinion

Reply:
ok